data/cameras.xml

```diff
@@ -12742,12 +12742,12 @@
 			</ColorMatrix>
 		</ColorMatrices>
 	</Camera>
-	<Camera make="SAMSUNG" model="EK-GN120" decoder_version="3" supported="no-samples">
-		<ID make="Samsung" model="EK-GN120">Samsung EK-GN120</ID>
+	<Camera make="SAMSUNG" model="EK-GN120" decoder_version="3">
+		<ID make="Samsung" model="EK-GN120">Samsung Galaxy NX</ID>
```

Comment:
I'm not really sure this kind of normalization is common here.
I think let's not do that.

Reply:
Well, it's officially called the Galaxy NX, but this is not a hill for me to die on :)

Reply:
Oh wait, i misread the change as-if the new maker was non-`Samsung`. Let's leave this in. Sorry.